docs/src/literate/falsesharing/falsesharing.jl

```diff
@@ -30,11 +30,11 @@ data = rand(1_000_000 * nthreads());
 #
 # A common, manual implementation of this idea might look like this:
 
-using OhMyThreads: @spawn, chunks
+using OhMyThreads: @spawn, index_chunks
 
 function parallel_sum_falsesharing(data; nchunks = nthreads())
     psums = zeros(eltype(data), nchunks)
-    @sync for (c, idcs) in enumerate(chunks(data; n = nchunks))
+    @sync for (c, idcs) in enumerate(index_chunks(data; n = nchunks))
         @spawn begin
             for i in idcs
                 psums[c] += data[i]
```

Comment:
Many (most?) of the examples could be rewritten to actually make use of the new default output of `chunks`, e.g.
```suggestion
    @sync for (c, data_chunk) in enumerate(chunks(data; n = nchunks))
        @spawn begin
            for d in data_chunk
                psums[c] += d
```
but perhaps easier to do in a separate PR.

Reply:
Yeah, let's do this in a separate PR.